support-pie/donate.html

```diff
@@ -7,6 +7,21 @@
 
 <div class="container" id="support">
 	<div class="row">
+		<h2>How you can help</h2>
+		<div class="col-md-8">
+			<p>Use our <a href="http://smile.amazon.com/ch/47-4554504"
+				target="_blank">AmazonSmile link</a> to support us! Once you sign up using our link, you can shop at
+				<a href="http://smile.amazon.com" target="_blank">smile.amazon.com</a> just like you would normally at amazon.com,
+				and we will receive 0.5% of the cost of any eligible purchases you make.
+			</p>
+		</div>
+		<div class="col-md-4" id="paying">
```

Comment:
We probably don't want 2 instances of the same id (`paying`) on the same page. Maybe use Bootstrap's `text-center` class instead?

http://getbootstrap.com/css/#type-alignment